packages/react-storage/src/index.ts

```diff
@@ -10,4 +10,5 @@ export {
   FilePickerProps,
   FileListHeaderProps,
   FileListFooterProps,
+  useSortHandler,
```

Comment:
Anything that gets exported here becomes a public export - does this hook need to be?

Reply:
With how I set up the table, this hook is used to help sort the data outside of the table. I'm not sure if this the right way to do that though. It is set up as a hook to move this code in a specific place for this review

I guess firstly, do you think having it sort outside of the table is still the approach we should take?